config/index.js

```diff
@@ -1,5 +1,54 @@
+const dotenv = require('dotenv').config({ path: `${__dirname}/.env` });
+
 const ENVIRONMENT = process.env.NODE_ENV || 'development';
 
 const configFile = `./${ENVIRONMENT}`;
 
-module.exports = require(configFile).config;
+const isObject = variable => {
+  return variable instanceof Object;
+};
+
+/*
+ * Deep copy of source object into tarjet object.
+ * It does not overwrite properties.
+*/
+const assignObject = (target, source) => {
```

Comment:
I actually don't remember what we agreed the past friday, are we going to leave this here or move it to `app/helpers/` folder? I would go with the second option.

Reply:
Second option :+1:

Reply:
If I could choose, I'd leave it here 😇